templates/example_flowsom_clustering.ipynb

```diff
@@ -47,9 +47,13 @@
    "outputs": [],
```

Comment:
<p>I think we should separate these directories into those that only have to be run once for preprocessing, and those that will be different for different combinations of parameters. </p><p> </p><p>For example, base_dir, tiff_dir, seg_dir, preprocessed_dir all belong to the first, subset, som, and consensus clustered belong to the second. </p><p> </p><p>For the first set of directories, we can auto-generate them based off the directory structure we assume people will have from running earlier parts of the pipeline. </p><p> </p><p>For the second set of directories, we should set a single name for that "experiment", like default_som_results. Then within that single folder, we'll auto-generate the subset_dir, som_clutered_dir, and consensus_clutered dir (and potentially others) in a known spot. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>